### ISSUE TITLE
Dropbox folder setting persists when switching back to app folder

### ISSUE THREAD

**Issue by zanderling** [keeweb/keeweb#697]
Using Dropbox as storage backend, I first had "Link the app to" set to "Full Dropbox or any folder", and then configured a custom app folder, say "Crypted".
Later, I switched back to the default setting "App folder (Apps/Keeweb)", only to find that Keeweb then stored the files not in "Apps/Keeweb" as expected, but concatenated the previously set custom app folder to this path, thus saving files to "Apps/Keeweb/Crypted". 
This behavior is unexpected to the user because the configured custom app folder (here "Crypted") is not visible anymore in the Settings UI as soon as "Link the app to" is set to "App folder (Apps/Keeweb)", but is still used.
Thank you for Keeweb! :)

**Comment by antelle** [comment]
This setting should be set before you open files. When you open files with one setting and change it later, it's very hard to remap existing files to a new folder, and it's kinda edge-case.

**Comment by zanderling** [comment]
Absolutely, I agree, I shouldn't have brought there program into that situation. But after I did, the settings UI was confusing, as it showed "App folder (Apps/Keeweb)" but then Keeweb appended the custom app folder, which was invisible at this point.
